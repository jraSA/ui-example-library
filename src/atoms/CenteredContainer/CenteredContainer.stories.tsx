import React from 'react';
import { Meta } from '@storybook/react/types-6-0';
import CenteredContainer from '.';
import Text from '../Text';

export default {
    title: 'Atoms/CenteredContainer',
    component: CenteredContainer,
} as Meta;

const Template = () => <CenteredContainer><Text>My text</Text></CenteredContainer>;

export const CenteredContainerComponent = Template.bind({});
