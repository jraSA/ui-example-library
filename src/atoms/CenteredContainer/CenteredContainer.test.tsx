import React from 'react';
import renderer from 'react-test-renderer';
import CenteredContainer from '.';

describe('Render test', () => {
    const wrapper = renderer.create(<CenteredContainer><div>foo-boo</div></CenteredContainer>);

    test('WHEN I render the component THEN it should match the snapshot', () => {
        expect(wrapper).toMatchSnapshot();
    });
});
