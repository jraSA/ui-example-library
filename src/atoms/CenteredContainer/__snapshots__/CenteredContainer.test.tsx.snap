// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render test WHEN I render the component THEN it should match the snapshot 1`] = `
<div
  className="sc-bdfBwQ ixcnvM"
>
  <div>
    foo-boo
  </div>
</div>
`;
